medicines/web/src/services/search-query-normalizer.ts

```diff
@@ -28,7 +28,7 @@ const addNormalizedProductLicenses = (q: string): string => {
 };
 
 const splitByNonSearchableCharacters = (query: string) =>
-  query.split(/(?:[,+\-!(){}\[\]^~*?:\/]|\s+)/gi);
+  query.split(/(?:[,+\-!(){}\[\]^~*?:%\/]|\s+)/gi);
```

Comment:
Is there not a way we can encode these characters for Azure?

If people are really searching for "100%awesome thing", then we probably don't want to.

Reply:
The current approach is to remove these symbols by splitting on them. I played around with encoding them instead and it seemed tricky to do this whilst constructing our fuzzy search from the broken up search terms. I wanted to get this quick fix in, but I agree it would be better to preserve the special characters used in the search by encoding them, instead of clearing them out. I'll create a card for this.